src/cpp/utils.cpp

```diff
@@ -0,0 +1,33 @@
+#include "utils.h"
+
+#include <fstream>
+#include <iomanip>
+
+namespace effil {
+
+Logger getLogger() {
+    static std::mutex lock;
+    static const char* logFile = getenv("EFFIL_LOG");
+
+    if (logFile == nullptr) {
+        static std::ostream devNull(0);
+        return Logger(devNull, lock);
+    }
+    else if (strcmp(logFile, "term") == 0) {
+        return Logger(std::cout, lock);
+    }
+    else {
+        static std::ofstream fileStream(logFile);
+        return Logger(fileStream, lock);
+    }
+}
+
+std::string getCurrentTime() {
+    const auto currTime = std::time(nullptr);
```

Comment:
Better use [`steady_clock::now()`](https://ru.cppreference.com/w/cpp/chrono/steady_clock).
It would be much shorter and cleaner.

Reply:
it will not be shorter because we still have to convert it to `time_t` and to `std::tm` after that.
Now it looks like that:
`const auto currTime = std::time(nullptr);`
using `chrono` it will be looks like that:
```
const auto now = std::chrono::system_clock::now();
const auto timet = std::chrono::system_clock::to_time_t(now);
```
Do you know another way?